docs/ops.html

```diff
@@ -4281,6 +4281,32 @@ <h4 class="anchor-heading"><a id="transaction_protocol_performance" class="ancho
   <li><code>add.partitions.to.txn.retry.backoff.max.ms</code></li>
 </ul>
 
+  <h3 class="anchor-heading"><a id="eligible_leader_replicas" class="anchor-link"></a><a href="#eligible_leader_replicas">6.12 Eligible Leader Replicas</a></h3>
+
+  <h4 class="anchor-heading"><a id="eligible_leader_replicas_overview" class="anchor-link"></a><a href="#eligible_leader_replicas_overview">Overview</a></h4>
+
+  <p>Starting from Apache Kafka 4.0, Eligible Leader Replicas (<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-966%3A+Eligible+Leader+Replicas">KIP-966 Part 1</a>)
+    is available for the users to enhance their data safety. As the "strict min ISR" rule has been generally applied, which means the high watermark for the data partition can't advance
```

Comment:
I think "an improvement to Kafka replication" would be better than "enhance their data safety"